governance/votes/Executive vote - April 16, 2021.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Template - [Executive Vote] Liquidations 2.0 Contracts, Increase Emergency Shutdown Threshold  - April 16, 2021
+---
+# [Executive Proposal] Liquidations 2.0 Contracts, Increase Emergency Shutdown Threshold  - April 16, 2021
+The Governance Facilitator(s) have placed an Executive Vote into the voting system on behalf of the Maker Governance community. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+### Authorize Liquidations 2.0 Contracts
+
+Given the passing of this [executive](https://vote.makerdao.com/executive/6058cea1efe023001bfeb313?network=mainnet#proposal-detail) the following contracts will be authorized to implement Liquidations 2.0:
```

Comment:
I wouldn't bother including addresses here. It's a level too deep for governance to engage with really. Just explain which new contracts are being authorized and which are being replaced.

Reply:
(and what they do, in simple terms)

Reply:
Awesome, decided to link to the Rocket,Chat channel so people can hunt them if they want. Not married to the idea if we wanna take it out

Reply:
Not sure, it would be good to link, chat probably isn't the best target.